docs/sources/developers/kinds/maturity.md

```diff
@@ -0,0 +1,403 @@
+| **Stage comms**              | Beta : we have 2 types of users that we want to target for this phase:- Users that we have already had contact with, regarding as code. This includes all customers that have requested the feature and all customers we have talked to in the past months - Campaign-recruited users: Advertise the functionality (blog post? Intercom?) and explain to customers how to enroll in this Beta phase.  - Customer expectations to be aligned to the [Beta](https://docs.google.com/document/d/1lqp0hALax2PT7jSObsX52EbQmIDFnLFMqIbBrJ4EYCE/edit#heading=h.ppvlnoblvlqd) phase int hte release definition document     |
+
+
+### Milestone 4 - Mature {#mature}
+
+###
+| **Goal**                     | The operational transition path for the Kind is battle-tested and reliable.                                                                                                                                                                                                                                                               |
+|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Reached when**             | A transition path from the pre-schematized world has been created and tested.<br/>Any feature flags guarding schema-centric behavior (e.g. generic object storage instead of legacy SQL storage for the Kind) are enabled by default or removed                                                                                           |
+| **Common hurdles**           | (We don’t know yet!)                                                                                                                                                                                                                                                                                                                      |
+| **Public-facing guarantees** | Backward compatibility for newer versions with previous schema versions (starting from open beta, 1.0 version). General support agreement for our GA features                                                                                                                                                                             |
+| **User-facing stage**        | Beta - open: anyone can use schemas, the functionality is marked asd beta.<br/> - Define metrics for Beta to GA transition (adoption? Number of change requests and bug opened trending down? ) Or should we skip open beta? All suggestions welcomeGA: the schema is launched and can be used as part of the Grafana toolkit for as-code |
+
+
```

Comment:
```suggestion
```

i think we should omit these for now - the definition of a maturity stage is an important thing to have formal agreement on by our arch board